ibis/backends/oracle/compiler.py

```diff
@@ -227,6 +227,14 @@ def visit_RandomScalar(self, op):
         # Not using FuncGen here because of dotted function call
         return sg.func("dbms_random.value")
 
+    def visit_RandomUUID(self, op):
```

Comment:
@jcrist There is no builtin `uuid` in oracle, I used the `sys_guid` and reformat it into a uuid format, but not sure if it is ok. since it a not a real uuid, it fails the `version == 4` test.

Reply:
If there's no `uuid`, then there's no `uuid` -- this can remain unimplemented for the Oracle backend unless we are very confident that the semantics match other implementations.